schema/software/flavor.json

```diff
@@ -31,6 +31,20 @@
             "items": {
                 "$ref": "../workflow/unit/input/_inputItemId.json"
             }
+        },
+        "contextProviders": {
+            "type": "array",
+            "items": {
+                "description": "render context provider names",
+                "$ref": "../workflow/unit/runtime/_runtime_item_name_object.json"
+            }
+        },
+        "templateParameters": {
```

Comment:
We need to adjust the logic for rendering the templates in the web application after this - where is that done? or will be done?

Reply:
will be done after all the new templates are created